src/commonmark-react-renderer.js

```diff
@@ -21,16 +21,18 @@ var defaultRenderers = {
 
     List: function List(props) {
         var tag = props.type === 'Bullet' ? 'ul' : 'ol';
-        return createElement(tag, (
-            props.start !== null &&
-            props.start !== 1 &&
-            { start: props.start.toString() } || null
-        ), props.children);
+        var attrs = { key: props.key };
+
+        if (props.start !== null && props.start !== 1) {
+            assign(attrs, { start: props.start.toString() });
```

Comment:
Not 100% sure it would cause issues:

```
2. Foo
3. Bar

2. Foo
3. Bar
```

Might be this gets wrapped in different parents, cant check right now